src/styles/settings/_colors.scss

```diff
@@ -13,4 +13,5 @@ $light-blue: #99ddff;
 $ice-blue: #e5f5ff;
 $red: #ea1c24;
 $muted-red: #c3645c;
-$orange: #dbad54
\ No newline at end of file
+$orange: #dbad54;
```

Comment:
I feel like making a joke about waking up in the land of Oz with more colors.... :rainbow: